libbeat/docs/tab-widgets/stuff.asciidoc

```diff
@@ -68,35 +65,19 @@
 
 [role="tab"]:hover::before,
 [role="tab"]:focus::before {
-  border-color: hsl(20, 96%, 48%);
+  border-color: hsl(218, 96%, 48%);
 }
 
 [role="tabpanel"] {
   position: relative;
   z-index: 2;
-  padding: 0.5em 0.5em 0.7em;
+  padding: 1em;
```

Comment:
I increased the padding inside the tab panel. Everything felt a little too compact to me before. Feel free to revert this if you disagree.